src/mmw/js/src/modeling/controllers.js

```diff
@@ -114,15 +114,37 @@ var ModelingController = {
             project = App.currentProject;
             project.set('model_package', modelPackage);
             itsiResetProject(project);
+            App.getMapView().addSidebarToggleControl();
         } else {
             var lock = $.Deferred();
-            project = makeProject(modelPackage, lock);
+            project = new models.ProjectModel({
+                name: 'Untitled Project',
+                created_at: Date.now(),
+                area_of_interest: App.map.get('areaOfInterest'),
+                area_of_interest_name: App.map.get('areaOfInterestName'),
+                model_package: modelPackage,
+                scenarios: new models.ScenariosCollection()
+            });
+
             App.currentProject = project;
+            lock.resolve();
+
+            App.getAnalyzeCollection().findWhere({taskName:'analyze'}).fetchAnalysisIfNeeded()
```

Comment:
Since `App.getAnalyzeCollection().findWhere({ taskName: 'analyze' })` is used multiple times, consider assigning it to a variable like `landSoilAnalyzeTask`.